frontend/src/views/admin/matching/index.tsx

```diff
@@ -1,14 +1,76 @@
 import React from "react";
+
 import { ContentArea } from "../../../components/layout";
 import { useSelector } from "react-redux";
 import { activeSessionSelector, fetchPostings } from "../../../api/actions";
 import { useThunkDispatch } from "../../../libs/thunk-dispatch";
-import { GroupedOverview } from "./grouped-overview";
+import { round } from "../../../libs/utils";
+
+import {
+    applicationsSelector,
+    assignmentsSelector,
+    positionsSelector,
+    applicantsSelector,
+} from "../../../api/actions";
+import { Assignment, Application, Applicant } from "../../../api/defs/types";
+
+import {
+    matchesSelector,
+    guaranteesSelector,
+    batchUpsertMatches,
+} from "./actions";
+import { PositionSummary, ApplicantSummary, Match } from "./types";
+
+import { PositionList } from "./position-list";
+import { ApplicantView } from "./applicant-view";
+
+import {
+    ImportMatchingDataButton,
+    ImportGuaranteesButton,
+    ExportMatchingDataButton,
+} from "./import-export";
+import { FinalizeChangesButton } from "./finalize-changes";
+
+function GetNewestApplication(
+    applicant: Applicant,
+    applications: Application[]
+) {
+    const matchingApplications = applications.filter(
+        (application) => application.applicant.id === applicant.id
+    );
+
+    if (matchingApplications.length === 0) {
+        return null;
+    }
+
+    matchingApplications.sort((a, b) => {
```

Comment:
`.sort()` should default to exactly the sort function you specified (except I believe `===` is replaced with `==`. You don't need to specify a function unless you're doing a different type of sort.